Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py

```diff
@@ -0,0 +1,416 @@
+        """
+        data = {"request_data": {"filters": search_params, "search_to": 100}}
+        headers = self._headers
+
+        response = self._http_request('POST', '/assets/get_assets_internet_exposure/',
+                                      json_data=data, headers=headers)
+
+        return response
+
+    def getassetinternetexposure_request(self, asm_id_list):
+        """Get internet exposure asset details using the '/assets/get_asset_internet_exposure/' endpoint.
+
+        Args:
+            asm_id_list (list): single attack surface management id in list format.
+
+        Returns:
+            dict: dict containing information on an internet exposure asset.
+        """
+        data = {"request_data": {"asm_id_list": asm_id_list}}
+        headers = self._headers
+
+        response = self._http_request('POST', '/assets/get_asset_internet_exposure/',
+                                      json_data=data, headers=headers)
+
+        return response
+
+
+def getexternalservices_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    asm-getexternalservices command: Returns list of external services.
+
+    Args:
+        client (Client): CortexAttackSurfaceManagment client to use.
+        args (dict): all command arguments, usually passed from ``demisto.args()``.
+            ``args['ip_address']`` IP Address to search on.
+            ``args['domain']`` Domain to search on.
+            ``args['is_active']`` If the service active or not.
+            ``args['discovery_type']`` how service was discovered.
+
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains external services.
+    """
+    ip_address = args.get('ip_address')
+    domain = args.get('domain')
+    is_active = args.get('is_active')
+    discovery_type = args.get('discovery_type')
+    # create list of search parameters or pass empty list.
+    search_params = []
+    if ip_address:
+        search_params.append({"field": "ip_address", "operator": "eq", "value": ip_address})
+    if domain:
+        search_params.append({"field": "domain", "operator": "contains", "value": domain})
+    if is_active:
+        search_params.append({"field": "is_active", "operator": "in", "value": [is_active]})
+    if discovery_type:
+        search_params.append({"field": "discovery_type", "operator": "in", "value": [discovery_type]})
+
+    response = client.getexternalservices_request(search_params)
+    parsed = response['reply']['external_services']
+    markdown = tableToMarkdown('External Services', parsed, removeNull=True)
+    command_results = CommandResults(
+        outputs_prefix='ASM.GetExternalServices',
+        outputs_key_field='service_id',
+        outputs=parsed,
+        raw_response=parsed,
+        readable_output=markdown
+    )
+
+    return command_results
+
+
+def getexternalservice_command(client: Client, args: Dict[str, Any]) -> CommandResults:
```

Comment:
[command name changes](https://github.com/demisto/content/pull/21226/commits/b7d6ed53f22c357ce4d4be0e10cb1c4702f1f7d7)

Reply:
Yes, but my suggestion was to add the word 'details' to the command name, because as I understood this command provides more details about the service compared to the 'asm-get-external-services' command. 
So my suggestion was to change the name to - 'asm-get-external-service-details', and in accordance to change the - outputs_prefix to 'ASM.ExternalServiceDetails'. 
@andrew-paloalto - would appreciate your opinion here.